### PR TITLE
fix junit format for reports

### PR DESCRIPTION
**Pull request by albfan**
**Fixes:** Issue #4373

**Relates to:** PR #4376

## Solution/Idea

adapt junit report reportportal format

## Proposed changes

post process xml junit to adapt xml to reportportal format

### CODE REVIEW

**Comment by albfan** [comment]
**NOTE** Windows still not tested

File: images/build-e2e/lib/darwin/filter.xsl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<xsl:stylesheet version="1.0"
```
Comment (adrianriobo) [inline images/build-e2e/lib/darwin/filter.xsl]: Are those xsls files exactly the same? If that is the case we need to have it only once and add it from the Containerfile

**Review by adrianriobo — COMMENTED** [review]
Also need to understand the context and the cause of this change on the commit log. (Also if possible prefix the commit log with [QE])

File: images/build-e2e/lib/darwin/filter.xsl (hunk 1 above)
Comment (albfan) [reply images/build-e2e/lib/darwin/filter.xsl]: Ok, I was rounding the idea. Will rework the PR

**Comment by albfan** [comment]
Doing test on reportportal, only content on failure tag is use to fill  stacktrace and logs

Our reason on a failure test is on message attribute from failure node, so this transformation remove message attribute and set as failure node content

File: images/build-integration/Containerfile [hunk 2]
```diff
@@ -21,5 +21,6 @@ ENV ASSETS_FOLDER=/opt/crc \
     ARCH=${ARCH}
 
 COPY --from=builder /workspace/out/${OS}-${ARCH} ${ASSETS_FOLDER}/bin
+COPY images/build-e2e/common/* ${ASSETS_FOLDER}/
```
Comment (adrianriobo) [inline images/build-integration/Containerfile]: This is using the wrong path no?

It should be `images/build-integration/common/*`

Comment (albfan) [reply images/build-integration/Containerfile]: ops

**Review by adrianriobo — APPROVED** [review]

**Pull request merged**